mappings/net/minecraft/entity/mob/EndermanEntity.mapping

```diff
@@ -6,7 +6,7 @@ CLASS net/minecraft/class_1560 net/minecraft/entity/mob/EndermanEntity
 	FIELD field_7255 ANGRY Lnet/minecraft/class_2940;
 	FIELD field_7256 ATTACKING_SPEED_BOOST_UUID Ljava/util/UUID;
 	FIELD field_7257 CARRIED_BLOCK Lnet/minecraft/class_2940;
-	METHOD method_7024 teleport (DDD)Z
+	METHOD method_7024 teleportTo (DDD)Z
```

Comment:
Is this `to` suffix really the right way to go about this conflict? I would prefer to name it something like `endermanTeleport`, as (iirc) the function does other specific things to do with endermen, like play the teleport sound.

Reply:
Named it `teleportTo` to match another method named teleportTo, which teleports to an entity instead of coordinates